### ISSUE TITLE
ranges (first..last) expect both sides to be integers, got: nil..nil

### ISSUE THREAD

**Issue by acravenho** [poanetwork/blockscout#1411]
Receiving this error on Kovan after the latest deployment. 

```
2019-02-05T17:48:34.820 [error] Task #PID<0.6154.3> started from Indexer.TokenTransfer.Uncataloged.Worker terminating
** (ArgumentError) ranges (first..last) expect both sides to be integers, got: nil..nil
    (elixir) lib/range.ex:55: Range.new/2
    (indexer) lib/indexer/block/catchup/fetcher.ex:352: anonymous fn/2 in Indexer.Block.Catchup.Fetcher.push_front/1
    (elixir) lib/enum.ex:3281: Enumerable.List.reduce/3
    (elixir) lib/enum.ex:1968: Enum.reduce_while/3
    (elixir) lib/task/supervised.ex:89: Task.Supervised.do_apply/2
    (elixir) lib/task/supervised.ex:38: Task.Supervised.reply/5
    (stdlib) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
Function: &Indexer.Block.Catchup.Fetcher.push_front/1
    Args: [[nil, 8994229, 8994224, 8994213, 8856521, 8856513]]
```

**Comment by acravenho** [comment]
This bug was caused by a `dropped/replaced` transaction without a block number. To find this transaction run 

```
SELECT DISTINCT t.block_number, encode(t.hash::bytea, 'hex')
FROM logs AS l
JOIN transactions AS t
ON t.hash = l.transaction_hash
LEFT JOIN token_transfers AS tf
ON tf.transaction_hash = l.transaction_hash
AND tf.log_index = l.index
WHERE l.first_topic = '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'
AND tf.transaction_hash IS NULL
AND tf.log_index IS NULL
```

**Comment by acravenho** [comment]
Closed by #1412
